x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/CsvTests.java

```diff
@@ -425,7 +425,7 @@ private static CsvTestsDataLoader.TestsDataset testsDataset(LogicalPlan parsed)
     }
 
     private static TestPhysicalOperationProviders testOperationProviders(CsvTestsDataLoader.TestsDataset dataset) throws Exception {
-        var testData = loadPageFromCsv(CsvTests.class.getResource("/" + dataset.dataFileName()), dataset.typeMapping());
```

Comment:
nit: Rather than duplicating the usage of the `/data` prefix here and in `CsvTestDataLoader.load()`, maybe we should stuff that into the `dataFileName()` method, so the file location logic is contained in one spot?